install/postgres/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu
+
+echo "Postgres version: $POSTGRES_VERSION"
+echo "Postgres cluster: $POSTGRES_CLUSTER"
+echo
+echo "Running database in foreground..."
+
+exec \
+	pg_ctlcluster "$POSTGRES_VERSION" "$POSTGRES_CLUSTER" \
+	start --foreground --stdlog
```

Comment:
We'll want to verify the logging for everything is something that can be picked up with `docker logs` for an end-user.

Reply:
If we want things in `docker logs` we should remove `--stdlog` and it'll appear in stdout, and thus `docker logs`.